src/index.js

```diff
@@ -118,6 +123,16 @@ class Anvil {
     )
   }
 
+  getEtchPacket ({ variables, responseQuery, query }) {
```

Comment:
So, are we or are we not allowing the `query` param? I think it's probably unnecessary...but you could also leave it in here as-is and it would be undocumented...but perhaps put a comment that says we are leaving it undocumented on purpose if so?

Reply:
Going to take the `query` param out since it's unnecessary.